### ISSUE TITLE
Vagrantise the Erls Releases

### ISSUE THREAD

**Issue by dineshkummarc** [ProcessManagement/ProcessManagement#27]
Vagrantise the Erls Releases

Use Vagrant
Use Puppetlabs for silent install
Silent install DotNet 4 .0/4 .5
Silent install IIS
Silent install WebDeployer for DotNet
Silent install MSSQL Server
Silent install Visual Studio

**Comment by denno-secqtinstien** [comment]
are we strong enough to support Vagrant like stuff?

**Comment by denno-secqtinstien** [comment]
are we strong enough to support Vagrant like stuff? i think this must have to be with v4.5 onwards. so lease mark it in future.

**Comment by dineshkummarc** [comment]
not yet but the future releases.

**Comment by dineshkummarc** [comment]
as currently we are working on v4.0 like stuff so would be moving it to future like stuff.

**Comment by dineshkummarc** [comment]
found some of the repositories which can help us implementing vagrant using dotnet technologies.

1. https://github.com/VagrantPlusPlus/vagrant_sql_server_express
2. https://github.com/VagrantPlusPlus/vagrant-mssql-express
3. https://github.com/VagrantPlusPlus/vagrantfile-windows-iis-mssql
4. https://github.com/VagrantPlusPlus/vagrant_sql_server_express
5. https://github.com/VagrantPlusPlus/vagrant-mono-aspnetvnext
6. https://github.com/VagrantPlusPlus/puppet-dotnet

**Comment by dineshkummarc** [comment]
I have created repositories -> https://github.com/ProcessManagement/DevOPSLAMP, https://github.com/ProcessManagement/DevOPS and https://github.com/ProcessManagement/DevOPSJAVA to track the progresses related to vagrant.

cc/ @denno-secqtinstien , @Neha-choudhary

**Comment by Neha-choudhary** [comment]
thank you @dineshkummarc for the info !

**Comment by Neha-choudhary** [comment]
I see a couple of new Repositories in VagrantPLUSPLUS to have java, php configurations on MAC, Linux and windows. are you planning to experiment with these technologies (i mean MAC basebox) also?

**Comment by denno-secqtinstien** [comment]
good work @dineshkummarc

**Comment by dineshkummarc** [comment]
Yes @Neha-choudhary i am just trying my hands to dirty with Vagrant like stuff on MAC Machines along with Linux and windows.
